src/ViewHelpers/DisplayStudentProfileViewHelper.php

```diff
@@ -22,159 +22,14 @@ class="btn btn-primary btn-sm ' . $fieldName . 'EditButton editbutton"
         );
     }
 
-    public static function outputApplicant(CompleteApplicantEntity $applicant): string
+    public static function studentChecks(bool $check): string
```

Comment:
this method name doesnt really mean anything. I have no idea what this method does by reading its name. please come up with a better name